cloudflare/resource_cloudflare_logpush_job.go

```diff
@@ -100,11 +110,9 @@ func resourceCloudflareLogpushJobCreate(d *schema.ResourceData, meta interface{}
 	log.Printf("[DEBUG] Creating Cloudflare Logpush Job from struct: %+v", job)
 
 	j, err := client.CreateLogpushJob(d.Get("zone_id").(string), job)
-
 	if err != nil {
-		return fmt.Errorf("error creating logpush job")
+		return fmt.Errorf("error creating logpush job: %v", err)
```

Comment:
@jacobbednarz this is the most important change in this PR, currently this error is swallowed so we don't know why Logpush job was not created. I haven't tested this change yet, but I think this fix is obvious. I can remove other fixes from this PR (but I'm not convinced that `d.Set()` errors are safe to ignore). Anyway, I'll try testing build with this change in my environment to check what is the actual problem when creating this Logpush job and I'll get back to you.

Reply:
That’s fine, I don’t have any issues expanding this error like you’ve done. 

As I mentioned above, the `Set`s are the only thing that I’m questioning whether we’re adding any value as almost every other instance of it’s usage (including outside of this provider) do not include error handling like you’ve added and I’d be looking to find the root cause as opposed to just logging like we’ve done here.